### PR TITLE
Make it easier to get started 

### PR DESCRIPTION
**Pull request by Bytes-Explorer**
## Why are these changes needed?
Simplify introduction to data prep kit and getting started by new users based on feedback from hands on workshops

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,19 +2,16 @@
 Our project welcomes external contributions. If you have an itch, please feel
 free to scratch it.
 
-To contribute code or documentation, please submit a pull request. 
-
-A good way to familiarize yourself with the codebase and contribution process is
-to look for and tackle low-hanging fruit in the issues. 
-Before embarking on a more ambitious contribution, please quickly [get in touch](#communication) with us.
+To contribute code or documentation, please submit a pull request. You can get started with open issues with the label - good first issue.
```
Comment (agoyal26) [inline CONTRIBUTING.md:5]: To contribute code or documentation, please submit a pull request. You can get involved by working on our open issues labeled 'good first issue'.

File: README.md [hunk 2]
```diff
@@ -9,40 +9,34 @@
 <?  [![GitHub Pull Requests](https://img.shields.io/github/issues-pr/kylelobo/The-Documentation-Compendium.svg)](https://github.com/IBM/data-prep-kit/pulls) ?>
 </div> 
 
----
-
 Data Prep Kit is a community project to democratize and accelerate unstructured data preparation for LLM app developers. 
-With the explosive growth of LLM-enabled use cases, developers are faced with the enormous challenge of preparing use case-specific unstructured data to fine-tune or instruct-tune the LLMs.
+With the explosive growth of LLM-enabled use cases, developers are faced with the enormous challenge of preparing use case-specific unstructured data to fine-tune, instruct-tune the LLMs or build RAG applications.
```
Comment (agoyal26) [inline README.md]: With the explosive growth of LLM-enabled use cases, developers are faced with the enormous challenge of preparing use case-specific unstructured data to fine-tune and instruct-tune the LLMs or to build RAG applications for LLMs.

File: README.md [hunk 3]
```diff
@@ -9,40 +9,34 @@
...
 As the variety of use cases grows, so does the need to support:
```
Comment (agoyal26) [inline README.md]: As the variety of use cases grow, so does the need to support:

File: README.md [hunk 4]
```diff
@@ -94,34 +79,82 @@ Matrix below shows the the combination of modules and supported runtimes. All th
...
-For a new module to be added, a user can pick the right design based on the processing to be applied. More details [here](transforms). 
+For a new module to be added, a user can pick the right design based on the processing to be applied. More details [here](transforms).
+
+One can add new transforms by bringing in Python-based processing logic and using the Data Processing Library to build and contribute transforms. We have provided an [example transform](transforms/universal/noop) that can serve as a template to add new simple transforms. One can follow the step by step [tutorial](data-processing-lib/doc/simplest-transform-tutorial.md) to help you add your own new transform. For more details, we encourage the reader to go through the [architectural overview](../../data-processing-lib/doc/overview.md) for the details on [transforms](../../data-processing-lib/doc/transforms.md) and [runtimes](../../data-processing-lib/doc/transform-runtimes.md). More details on the data processing library are [here](data-processing-lib/doc/overview.md).
```
Comment (agoyal26) [inline README.md]: One can leverage Python-based processing logic and the Data Processing Library to easily build and contribute new transforms. We have provided a simple "noop" transform (transforms/universal/noop) as a template to get one started. Follow our step-by-step tutorial (data-processing-lib/doc/simplest-transform-tutorial.md) to learn the ropes.

For a deeper understanding of the library's architecture, its transforms, and available runtimes, we encourage the reader to consult the comprehensive overview document (../../data-processing-lib/doc/overview.md) alongside dedicated sections on transforms (../../data-processing-lib/doc/transforms.md) and runtimes (../../data-processing-lib/doc/transform-runtimes.md).

Additionally,  check out our video tutorial (https://www.youtube.com/watch?v=0WUMG6HIgMg) for a visual, example-driven guide on adding custom modules.

**Review by shivdeep-singh-ibm — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Bytes-Explorer) [reply CONTRIBUTING.md:5]: Updated

File: README.md (hunk 2 above)
Comment (Bytes-Explorer) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Bytes-Explorer) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (Bytes-Explorer) [reply README.md]: Done

**Comment by sujee** [comment]
May be merge all setup instructions together?

README.md around line 106

```bash
pip3 install  data-prep-toolkit-transforms-ray
pip3 install jupyterlab   ipykernel

## install custom kernel
python -m ipykernel install --user --name=data-prep-kit --display-name "dataprepkit"
```

**Comment by sujee** [comment]
Some comments on markdown for root README.md

**line 147**

there is a typo,  REMOVE space between `[]` and `()`  

` [overview document] (../../data-processing-lib/doc/overview.md)`

**line 139**

replace with : 

`[video tutorial](https://www.youtube.com/watch?v=0WUMG6HIgMg)`

**overall md**

Add a new line between segments.  For example

line 123, add a newline between the  code segment and text.

**Comment by sujee** [comment]
README.md

**line 110**

To test DPK is installed, try importing the data prep kit libraries in python CLI

```bash
## start python interpretter
$   python

# import DPK libraries
from data_processing_ray.runtime.ray import RayTransformLauncher
from data_processing.runtime.pure_python import PythonTransformLauncher
```

If there are no errors, you are good to go!

**can we print DPK version**

Is there way to print the version of DPK module?

**Pull request merged**